package.json

```diff
@@ -88,10 +88,13 @@
     "@nestjs/graphql": "^7.2.4",
     "@nestjs/platform-fastify": "^7.0.7",
     "@prisma/client": "^2.0.0-beta.5",
+    "@types/graphql-depth-limit": "^1.1.2",
```

Comment:
This dependency should be in `devDependencies`

Reply:
resolved!